doc/src/sgml/config.sgml

```diff
@@ -2145,6 +2145,34 @@ include_dir 'conf.d'
       </listitem>
      </varlistentry>
 
+     <varlistentry id="guc-max-total-memory" xreflabel="max_total_memory">
+      <term><varname>max_total_memory</varname> (<type>integer</type>)
+      <indexterm>
+       <primary><varname>max_total_memory</varname> configuration parameter</primary>
+      </indexterm>
+      </term>
+      <listitem>
+       <para>
+        Specifies a limit to the total amount of memory (MB) a database server
+        can allocate. (i.e. this is not a per user or per backend limit).
+        If unset or set to 0, it is disabled. max_total_memory must be larger than
+        shared_memory_size and should include an additional 2MB per connection.
+        </para>
+        <para>
+        Any memory request that would exhaust the limit will
+        be denied with an out of memory error, causing that process's current
+        query/transaction to fail. Further requests will not be allocated until
+        total memory used drops below the limit.
+        Memory requests from the postmaster are tracked,
+        but the postmaster is able to exceed the limit.
+        Total memory used (<varname>total_memory_used</varname>) is displayed in the
+        <link linkend="monitoring-pg-stat-memory-allocation-view">
+        <structname>pg_stat_memory_allocation</structname></link> view.
```

Comment:
This should be saying pg-stat-global-memory-allocation, since the total isn't in the pg-stat-memory-allocation view.